recipes/pennies/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pennies" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5044bfd6009dc4d6d14ed8d8355d0cc4b9cb4aafad105b80b0c481f8beacdc15
+  patches:
+    - 0001-fix-imports.patch
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
```

Comment:
I think you only need `python` and `pip` in the `host` section since the rest are not required to build the package. Also `pip` is listed twice.